src/styles/blocks/calanders.scss

```diff
@@ -0,0 +1,22 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gaps;
+  padding: 0 $paddings;
+
+  &__day {
+    @extend %flexik;
+    @include square;
+
+    border: 1px solid #000;
+    background-color: #eee;
+
+    transition: 0.5s;
+
```

Comment:
I think it is better to declare what keyword you want to set transition, for example, color, transform, or all

Reply:
good comment by @Alireza-Mohammadhossein 
@miftaha if you don't specify - browser runs transition for all the props using excessive resources.